### PR TITLE
V3.8.2 enable resize inspector preview area height

### PR DESCRIPTION
**Pull request by arsen2010**
Re: #https://github.com/cocos/3d-tasks/issues/17386

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/inspector/assets/fbx/index.js [hunk 1]
```diff
@@ -97,9 +98,13 @@ const Elements = {
...
         update() {
             const panel = this;
-            Editor.Message.broadcast('fbx-inspector:change-tab', panel.activeTab);
             panel.$.tabPanel.setAttribute('src', Components[panel.activeTab]);
             panel.$.tabPanel.update(panel.assetList, panel.metaList);
+
+            // Delay, waiting for the fbx preview area initialization to complete
+            setTimeout(() => {
+                Editor.Message.broadcast('fbx-inspector:change-tab', panel.activeTab);
+            });
```
Comment (doommm) [inline editor/inspector/assets/fbx/index.js:107]: why using `setTimeout` here?

File: editor/inspector/contributions/asset.css [hunk 2]
```diff
@@ -101,11 +101,33 @@
...
+}
+
+:host > .container > .content > section > inspector-resize-preview[area="footer"] {
+    top: 0;
+}
+
+:host > .container > .content > .content-header,
+:host > .container > .content > .content-footer {
+    max-height: 80vh;
 }
```
Comment (doommm) [inline editor/inspector/contributions/asset.css:125]: avoid using unnecessary nested selector and it's better not to reply on specific DOM structure. make it clear, make it simple

File: editor/inspector/assets/fbx/index.js (hunk 1 above)
Comment (arsen2010) [reply editor/inspector/assets/fbx/index.js:107]: 因为 fbx 的头部区域也在初始化监听此广播事件，此为发送位置，需要等下监听位置初始化完成

File: editor/inspector/contributions/asset.css (hunk 2 above)
Comment (arsen2010) [reply editor/inspector/contributions/asset.css:125]: Specific structure selector can reduce false matches.

**Review by doommm — APPROVED** [review]

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**